src/server/replica.cc

```diff
@@ -355,6 +384,7 @@ error_code Replica::Greet() {
     master_context_.master_repl_id = param0;
     master_context_.dfly_session_id = param1;
     num_df_flows_ = param2;
+    io_buf.ConsumeInput(consumed);
```

Comment:
because of the bug in CheckRespIsSimpleReply we didnt see the error in the next call to this function.
Now after consuming the check passes